models/klaviyo__events.sql

```diff
@@ -6,7 +6,7 @@
             "field": "occurred_on",
             "data_type": "date"
         } if target.type == 'bigquery' else none,
-        incremental_strategy = 'merge',
+        incremental_strategy = 'merge' if target.type != 'snowflake' else 'delete+insert',
```

Comment:
Can you add an inline comment here explaining the purpose of this conditional? I remember we found the supporting documentation, but can see us forgetting or not understanding why we did this down the road and an inline comment would be helpful for us and customers.